model_to_omics_data_config.json

```diff
@@ -0,0 +1,170 @@
+{
+    "table_filters": {
+        "_comment": "Contains the filters to be applied to the tables, the filter_mode can be 'all'(all filters should be true) or 'any' (at least one of the filters should be true)",
+        "omics_sequencing": {
+            "_comment": "Gumbo table name 'omics_sequencing' is block key. Table name MUST exactly match database table name",
+            "exclusion_filters": {
+                "_comment": "Exclude rows where database column 'omics_sequencing.blacklist' has value 1(is True)",
+                "columns": ["blacklist"],
+                "blacklist": [1],
+                "filter_mode": "all"
+            },
+            "inclusion_filters": {
+                "_comment": "Include rows where db column 'omics_sequencing.expected_type' is rna,wgs or wes",
+                "columns": ["expected_type"],
+                "expected_type": ["rna", "wgs", "wes"],
+                "filter_mode": "any"
+            }
+        },
+        "omics_profile": {
+            "_comment": "Gumbo table name 'omics_profile' is block key. Table name MUST exactly match database table name",
+            "exclusion_filters": {
```

Comment:
Here we should also exclude profiles where the profile_source is "taiga". Those are legacy profiles that we don't have bams for